stable/airflow/templates/flower/flower-deployment.yaml

```diff
@@ -109,7 +109,12 @@ spec:
                && mkdir -p /home/airflow/.local/bin \
                && export PATH="/home/airflow/.local/bin:$PATH" \
                && echo "*** running flower..." \
+              {{- if .Values.flower.oauthDomains }}
+               && exec airflow flower --auth="{{ .Values.flower.oauthDomains | quote }}"
```

Comment:
this will cause a double quote it should be
`&& exec airflow flower --auth={{ .Values.flower.oauthDomains | quote }}`